### ISSUE TITLE
The application “Service Station Attendant” does not have permission to open

### ISSUE THREAD

**Issue by pkamb** [knurling/ServiceStation#19]
I right clicked a file or folder and selected a Service Station menu item. But it gives the error:

![Screen Shot 2020-04-19 at 12 34 16 AM](https://user-images.githubusercontent.com/342355/79682258-7c33ad00-81d5-11ea-8f17-40309737dff7.png)

 > The application “Service Station Attendant” does not have permission to open “FILE_OR_FOLDER.”

Why is this not working?

**Comment by pkamb** [comment]
Service Station is [Sandboxed](https://developer.apple.com/app-sandboxing/), which is a security requirement for all apps in the Mac App Store.

What this means in practice is that Service Station needs you to grant it access to your Mac's file system before it can open the files or folders you right click.

This is the functionality of the `System Setup > Finder Locations` picker:

![Screen Shot 2020-04-19 at 12 44 07 AM](https://user-images.githubusercontent.com/342355/79682467-6626ec00-81d7-11ea-8b3e-e7921a41d231.png)

Service Station will be able to open any files and folders that are contained within the directories you add to the Finder Locations list.

For best results, please add a major parent directory such as `/` or your Home folder. This will ensure that Service Station will have access to open any files you select.

**Comment by tjluoma** [comment]
I was able to solve this by giving Service Station Full Disk Access in System Preferences » Security & Privacy » Privacy » Full Disk Access.

Click the "+" under the right column and select the Service Station app. Allow it to be restarted (Big Sur) or manually restart it (Catalina) and voilà!

**Comment by rRuh** [comment]
I've tried all of these things, and after multiple restarts, still cannot get it working. I'm running macOS Big Sur 11.7.6 and the latest app store version of Service Station. Any ideas?

**Comment by gingerbeardman** [comment]
@rRuh 

- open system preferences
- go to Security & Privacy
- remove it completely from all the permissions boxes
    - click padlock and authenticate 
    - find the app
    - clicking the - minus button to remove
- close system preferences
- open system preferences
- go to Security & Privacy
- add the app again

follow these steps, don't try to take shortcuts

**Comment by rRuh** [comment]
Thanks @gingerbeardman !  The step I was missing was to close and reopen system preferences.

**Comment by Bytesmiths** [comment]
I am unable to make this fix in Ventura 13.7.6. The System Settings appear to have changed, with no apparent way to add arbitrary Applications to the list of those that can access all files.

I was able to add "Service Station" to the "Full Disk Access" list, but that didn't seem to do anything. I quit and restarted Service Station. I quit and restarted System Settings. I killed the process com.knurling.ServiceStation.Attendant, and it restarted itself.

Do I need to log out or re-boot for any of these actions to take effect is allowing "Service Station Attendant" to work?

**Comment by luckman212** [comment]
Just had to do this on macOS 26 Tahoe.

<img width="260" height="240" alt="Image" src="https://github.com/user-attachments/assets/bf83dca3-3c64-4810-ab72-355237d9f0bd" />

What fixed it was removing "Service Station Attendant" from Full Disk Access and then adding it back. The shortcut to get there (so you can drag it back into the Privacy prefs window) is

### `⌘Cmd` + `⇧Shift` + `G`

```
/Applications/Service Station.app/Contents/Library/LoginItems/Service Station Attendant.app
```
